pallets/im-online/src/lib.rs

```diff
@@ -352,6 +352,8 @@ decl_error! {
     }
 }
 
+type Session<T> = pallet_session::Module<T>;
+
```

Comment:
Let's rollback the changes made to this pallet. This has been copied from Parity's codebase and any changes we make here will make it harder to merge future upstream changes. Since none of these changes look very significant, I'll suggest either just forgetting bout them or creating a PR upstream.